dev/com.ibm.ws.security.wim.adapter.ldap/resources/com/ibm/ws/security/wim/adapter/ldap/resources/LdapUtilMessages.nlsprops

```diff
@@ -186,9 +186,9 @@ ENTITY_HAS_DESCENDENTS=CWIML4519E: The user registry operation could not be comp
 ENTITY_HAS_DESCENDENTS.explanation=The delete or rename function can only be performed on a leaf entity, that is, an entity without any descendants. The specified entity has descendants.
 ENTITY_HAS_DESCENDENTS.useraction=Delete all the  descendants of the entity first, then delete or rename the entity itself.
 
-ENTITY_TYPE_NOT_SUPPORTED=CWIML0510W: The user registry operation could not be completed. {0} is not valid entity type. Invoke the operation by specifying a valid entity type.
-ENTITY_TYPE_NOT_SUPPORTED.explanation=An operation has been requested on an entity, but that operation does not support the specified entity type. For example, if a repository GET operation is invoked for an Account entity, the operation fails.
-ENTITY_TYPE_NOT_SUPPORTED.useraction=Ensure that operations are invoked for supported entity types and remove any unsupported entity types from the input.
+ENTITY_TYPE_NOT_SUPPORTED=CWIML0510W: The entity type {0} specified is not a valid entity type for this operation.
```

Comment:
ENTITY_TYPE_NOT_SUPPORTED=CWIML0510W: The entity type {0} specified is not valid for this operation.

Reply:
Updated.